public/locales/en/panel-user.json

```diff
@@ -0,0 +1,22 @@
+{
+  "local-panels": {
+    "title": "Open Local Panel",
+    "select-panel-placeholder": "Select panel",
+    "aria-label": "Open local panel"
+  },
+  "web-panels": {
+    "title": "Open From URL",
+    "input": {
+      "title": "Title (optional)",
+      "placeholder": "Panel title (optional)"
+    },
+    "url": {
+      "title": "URL",
+      "placeholder": "Panel URL (for example, https://example.com/)",
+      "aria-label": "Open web panel"
+    }
+  },
+  "recently-opened-panels": {
+    "title": "Recently Opened URLs"
+  }
```

Comment:
I see that you named the key "recently opened panels", while it's technically just the URL ones

Should we really include all recently opened panels, and not just the web/URL ones? that would probably make more sense, to me

Poteantially out of the scope of this PR (although, you HAVE poked around in this part of the code so why not I guess), but maybe make an issue?